client/src/main/java/io/split/client/SplitClientConfig.java

```diff
@@ -91,6 +92,7 @@ public class SplitClientConfig {
     private final HashSet<String> _flagSetsFilter;
     private final int _invalidSets;
     private final CustomHeaderDecorator _customHeaderDecorator;
+    private final String _authScheme;
```

Comment:
Please create an enum for this property which accepts `kerberos`

Reply:
There is only one value for now, is it worth enum?

Reply:
ok, will do